server.py

```diff
@@ -10,13 +13,60 @@
     allow_methods=['*'],
     allow_headers=['*'],
 )
+FILE='urls.db'
+
 @app.get("/")
 def root():
-    return({ "message": "Welcome to the URL Shortener" })
+    return({ "message": "hello world" })
+
+@app.post("/create-url")
+def create_url(request : Request):
+    # process request into json
+    req_json = request.json()
+    # if link and alias are defined
+    if req_json["link"] and req_json["alias"]:
+        # try to insert url
+        result = insert_url(FILE, req_json["link"], req_json["alias"])
+        # if no duplicates
+        if result:
+            return({"message" : f"success: inserted link {req_json["link"]} as alias {req_json["alias"]}"})
```

Comment:
this way you can see the data cleanly that you inserted, it returns you what you entered and you can verify that it matches without extra wording
you dont want to add extra verbage since the debug logs will get bloated fast for no reason
```suggestion
            return({"url" : req_json["link"], "alias": req_json["alias"]})
```